src/credentials/providers/sharedCredentialsProvider.ts

```diff
@@ -20,6 +25,10 @@ const SHARED_CREDENTIAL_PROPERTIES = {
     ROLE_ARN: 'role_arn',
     SOURCE_PROFILE: 'source_profile',
     MFA_SERIAL: 'mfa_serial',
+    SSO_START_URL: 'sso_start_url',
+    SSO_REGION: 'sso_region',
+    SSO_ACCOUNT_ID: 'sso_account_id',
+    SSO_ROLE_NAME: 'sso_role_name',
```

Comment:
Out of scope for this PR, but will we ever support different degrees of capitalization for these fields? We have had issues in the past with people using things like `AWS_ACCESS_KEY_ID`, but it might also be moot since the JS SDK might not handle them?

Reply:
I think it might be better to suggest edits/corrections to their credentials/config file rather than try to accept an incorrect field name.